app/commands/exercise/export_solutions_to_zip_file.rb

```diff
@@ -0,0 +1,28 @@
+require 'zip'
+
+class Exercise::ExportSolutionsToZipFile
+  include Mandate
+
+  initialize_with :exercise
+
+  def call
+    file_stream = Zip::OutputStream.write_buffer do |zip|
+      solutions.each.with_index do |solution, idx|
+        solution.latest_iteration.files.each do |iteration_file|
+          zip.put_next_entry "#{idx}/#{iteration_file.filename}"
+          zip.print iteration_file.content.force_encoding("utf-8")
+        end
+      end
+    end
+
+    file_stream.rewind
```

Comment:
I always find it so weird that we have to manually do this!

Reply:
Yes!